### PR TITLE
ODM-12172: FCY and aFile - related changes

### PR DESCRIPTION
**Pull request by m-smazhevsky**
Screenshot 2024-11-25 at 12 02 08
Screenshot 2024-11-25 at 12 07 48
Screenshot 2024-11-25 at 12 02 47
Screenshot 2024-11-25 at 12 02 20

### CODE REVIEW

File: openapi/v1/job.yaml [hunk 1]
```diff
@@ -311,9 +321,16 @@ paths:
...
+        * `source` - file source with the following possible values: `S3`, `HTTP`, `LOCAL`.
+        * `dataLink` - a link to a file to import as an attachment.
+        The file will be associated with a study and will be searchable by its name and metadata.
+        * `metadataLink` - an optional URL of a metadata file to be used as the original metadata
+        for the created objects. The file must contain a single record describing the uploaded attachment.
+        * `studyAccession` - an accession of a study the file will be associated with.
+        * `dataClass` - file data class with the following possible values: `Bulk
         transcriptomics`, `Single-cell transcriptomics`, `Differential abundance (FC,
         pval, etc.)`, `Pathway analysis`, `Proteomics`, `Single-cell proteomics`, `Metabolomics`,
         `Lipidomics`, `Epigenomics`, `DNA methylation`, `Chemoinformatics`, `Imaging features`,
```
Comment (natalyatodorova) [inline openapi/v1/job.yaml:336]: I believe `Flow Cytometry (FCS)` data class should be also listed here

File: openapi/v1/flowCytometryUser.yaml [hunk 2]
```diff
@@ -2,6 +2,7 @@ openapi: 3.1.0
 info:
   description: |-
     This swagger page describes the flowCytometryUser API endpoints for ODM. These are typically used to find and retrieve flow cytometry data and metadata.
```
Comment (egor-alexandrov) [inline openapi/v1/flowCytometryUser.yaml:4]: Minor: while we're here, maybe it's a good time to unify this text for both flowCytometryCurator/flowCytometryUser  pages? There are some slight differences in them

Comment (m-smazhevsky) [reply openapi/v1/flowCytometryUser.yaml:4]: I didn't feel enough courage to do that, but now I feel. Will do after merging develop with Nikola's changes

Comment (m-smazhevsky) [reply openapi/v1/flowCytometryUser.yaml:4]: done

File: openapi/v1/job.yaml (hunk 1 above)
Comment (m-smazhevsky) [reply openapi/v1/job.yaml:336]: happend automatically

File: openapi/v1/flowCytometryUser.yaml (hunk 2 above)
Comment (m-smazhevsky) [reply openapi/v1/flowCytometryUser.yaml:4]: also decided to remove read the docs from endpoint descriptions

**Review by egor-alexandrov — APPROVED** [review]

**Review by natalyatodorova — APPROVED** [review]

**Pull request merged**